### PR TITLE
Russian language update

### PR DESCRIPTION
**Pull request by MasterGroosha**
New strings translated, except "terms" key.
Should this be translated? Can it be changed by end user?

### CODE REVIEW

File: config/locales/ru.yml [hunk 1]
```diff
@@ -517,10 +618,52 @@ ru:
...
+      <p>Our site, products and services are all directed to people who are at least 13 years old. If this server is in the USA, and you are under the age of 13, per the requirements of COPPA (<a href="https://en.wikipedia.org/wiki/Children%27s_Online_Privacy_Protection_Act">Children's Online Privacy Protection Act</a>) do not use this site.</p>
+      <h3 id="online">Online Privacy Policy Only</h3>
+      <p>This online privacy policy applies only to information collected through our site and not to information collected offline.</p>
+      <h3 id="consent">Your Consent</h3>
+      <p>By using our site, you consent to our web site privacy policy.</p>
+      <h3 id="changes">Changes to our Privacy Policy</h3>
+      <p>If we decide to change our privacy policy, we will post those changes on this page.</p>
+      <p>This document is CC-BY-SA. It was last updated May 31, 2013.</p>
+      <p>Originally adapted from the <a href="https://github.com/discourse/discourse">Discourse privacy policy</a>.</p>
+    title: "Условия обслуживания и политика конфиденциальности %{instance}"
```
Comment (ykzts) [inline config/locales/ru.yml]: `title: "Условия обслуживания и политика конфиденциальности %{instance}"` -> `title: Условия обслуживания и политика конфиденциальности %{instance}`

remove double quote

**Review by ykzts — CHANGES_REQUESTED** [review]

File: config/locales/ru.yml (hunk 1 above)
Comment (MasterGroosha) [reply config/locales/ru.yml]: Are you sure? English version has double quotes there: https://github.com/tootsuite/mastodon/blob/master/config/locales/en.yml#L692

Comment (ykzts) [reply config/locales/ru.yml]: YAML can not use "%" at the beginning of the character string following ":". Therefore, en.yml encloses it in double quotes.

Comment (MasterGroosha) [reply config/locales/ru.yml]: @ykzts Gotcha, fixed

**Review by ykzts — APPROVED** [review]

**Comment by ykzts** [comment]
[why?](https://travis-ci.org/tootsuite/mastodon/requests)

**Comment by MasterGroosha** [comment]
@ykzts I don't really know what "Abuse detected" means in terms of Travis CI.
I don't use it (and never used) and I did nothing there except pushing 2 commits you see above.

Why Travis shows this message is honestly out of my knowledge, sorry.

**Comment by ykzts** [comment]
No problem with other pull requests. Since it may be a problem specific to your account, can you contact support@travis-ci.com ?

**Comment by MasterGroosha** [comment]
@ykzts I've contacted Travis Support team, now waiting for reply.
I think I know what's the problem with Travis. According to [a similar issue](https://github.com/travis-ci/travis-ci/issues/7662) I'm "flagged" because of name changing. (I changed Github username a while ago and was previously signed to Travis with the old username).

Now, back to Mastodon and this PR. If Travis doesn't build with changed made by me, what should I do? Maybe someone else can use my changes and make a PR from themselves. Any other ideas?

**Comment by MasterGroosha** [comment]
Thank you, @Gargron!

**Pull request merged**